.drone.yml

```diff
@@ -55,11 +55,12 @@ name: feature-discussion
 
 steps:
   - name: Run unit tests
-    image: openjdk:11.0.2-jdk-oracle
+    image: maven:3.8.1-jdk-11
```

Comment:
you will also need corresponding change in Dockerfile for staging and prod deploy

Reply:
Ok, will make the change.